sqlmesh/core/model/decorator.py

```diff
@@ -38,6 +38,10 @@ def __init__(self, name: str, **kwargs: t.Any) -> None:
             )
             for call in calls
         ]
+        if "table_properties" in self.kwargs:
```

Comment:
are you sure this is needed? isn't this what was done before?

Reply:
What was done before?

Reply:
convert was called in the pydantic validator